src/App.tsx

```diff
@@ -1,23 +1,22 @@
-import SearchBar from "../src/components/SearchBar/SearchBar.tsx"
-import CountryTemperature from "../src/components/CountryTemperature/CountryTemperature.tsx"
-import DayWiseWeather from "../src/components/DayWiseWeather/DayWiseWeather.tsx"
-// import SearchBar from "../src/components/TitleBar"
-
-import './App.css'
+//components
+import SearchBar from "./components/NavBar";
```

Comment:
Change the name of this component to NavBar

Reply:
done